src/library/scala/Option.scala

```diff
@@ -116,14 +116,38 @@ sealed abstract class Option[+A] extends Product with Serializable {
   self =>
 
   /** Returns true if the option is $none, false otherwise.
+   *
+   * This is equivalent to:
+   * {{{
+   * option match {
+   *   case Some(_) => false
+   *   case None    => true
+   * }
+   * }}}
    */
   def isEmpty: Boolean
 
   /** Returns true if the option is an instance of $some, false otherwise.
+   *
+   * This is equivalent to:
+   * {{{
+   * option match {
+   *   case Some(_) => true
+   *   case None    => false
+   * }
+   * }}}
    */
   def isDefined: Boolean = !isEmpty
 
   /** Returns the option's value.
+   *
+   * This is equivalent to:
+   * {{{
+   * option match {
+   *   case Some(x) => x
+   *   case None    => throw new Exception
```

Comment:
Can you make it a `NoSuchElementException` here?

Reply:
I'm trying to keep things illustrative, rather than precise.

Reply:
Thanks, I'm going to use that line with my daughter.